### ISSUE TITLE
Reproduction on CIFAR-100 LT IF=100

### ISSUE THREAD

**Issue by rtu715** [val-iisc/DeiT-LT#5]
Hello authors, I have ran training on a single GPU and tested both the original batch size and an increased batch size (2X since given script runs on two GPUs). In all these cases, the training script does not yield a model with comparable accuracy as those reported in the paper. 

The results are the following:
```
BEST NUMBERS ----->
Overall / Head / Med / Tail
AVERAGE:  45.3  /  64.972  /  45.2  /  20.966
CLS    :  29.66  /  58.556  /  23.371  /  1.414
DIST   :  37.83  /  33.556  /  39.657  /  40.966
```

I wonder how these detailed results compare to yours, and moreover, what other items are necessary to reproduce the DeiT-LT results? I am happy to provide more details as well. Thanks!
